reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -8,6 +8,11 @@
 
 <header id="liveupdate-header">
   <h1 id="liveupdate-title">${thing.event.title}</h1>
+  % if thing.event.description:
+  <div id="liveupdate-description">
```

Comment:
yeah, at some point I should just add a helper that returns the doubly-wrapped markdown.  the text will be fairly small without it.